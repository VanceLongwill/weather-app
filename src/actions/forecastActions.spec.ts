import {
  fetchForecast,
  fetchForecastFail,
  fetchForecastSuccess,
} from './forecastActions'
describe('forecast actions', () => {
  it('action creators should match snapshot', () => {
    expect(fetchForecast('London')).toMatchSnapshot()
    expect(fetchForecastFail('some error')).toMatchSnapshot()
    expect(
      fetchForecastSuccess({ displayName: 'something' } as any)
    ).toMatchSnapshot()
  })
})
