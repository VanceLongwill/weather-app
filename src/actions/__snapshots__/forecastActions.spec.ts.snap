// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`forecast actions action creators should match snapshot 1`] = `
Object {
  "payload": Object {
    "city": "London",
  },
  "type": "FORECAST_FETCH",
}
`;

exports[`forecast actions action creators should match snapshot 2`] = `
Object {
  "payload": Object {
    "error": "some error",
  },
  "type": "FORECAST_FETCH_FAIL",
}
`;

exports[`forecast actions action creators should match snapshot 3`] = `
Object {
  "payload": Object {
    "result": Object {
      "displayName": "something",
    },
  },
  "type": "FORECAST_FETCH_SUCCESS",
}
`;
